src/components/Navigation.jsx

```diff
@@ -43,17 +42,21 @@ const Navigation = () => {
               className="flex flex-col items-center hover:!text-isa-yellow-100 text-3xl font-bold text-white ml-7 mr-7"
             >
               {item.name}
-              <div className="flex justify-center md:absolute bottom-5 h-1">
-                {pathname === item.name && (
-                  <FaMinus className="text-isa-yellow-200" />
-                )}
+              <div className="flex justify-center md:absolute md:mt-9">
+                <div
+                  className={
+                    pathname === item.name
+                      ? "bg-gradient-to-r from-isa-yellow-200 to-isa-yellow-100 px-4 py-1 rounded-full "
+                      : "hidden"
+                  }
+                ></div>
               </div>
             </Nav.Link>
           ))}
 
           <Nav.Link
             href="\"
-            className="flex justify-center items-center text-white text-3xl font-bold w-44 h-11 rounded-full bg-gradient-to-r from-isa-yellow-200 to-isa-yellow-100"
+            className="flex justify-center items-center text-white text-3xl font-bold w-44 h-11 rounded-full bg-gradient-to-r from-isa-yellow-200 to-isa-yellow-100 md:mt-0 mt-2"
```

Comment:
<img width="643" alt="image" src="https://github.com/acm-ucr/isa-website/assets/91030482/981328ce-f6e8-4e29-bd13-71708a3deb9c">

the join button looks a little misaligned here. what does the mobile responsiveness supposed to do here?